style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+html {
+  scroll-behavior: smooth;
+}
+body {
+  font-family: sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+#navbar {
+  display: flex;
+  justify-content: flex-end;
+  top: 0;
+  left: 0;
+  background-color: rgb(201, 77, 97);
+  font-size: 20px;
+}
+.navlink {
+  margin-right: 15px;
+  color: #f0f0f0;
+  padding: 18px;
+}
+#welcome-section {
+  background-color: rgb(228, 210, 220);
+  height: 100vh;
+  width: auto;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+h1 {
+  margin: 30px;
+  font-size: 50px;
+}
+
+h2 {
+  font-size: 30px;
+}
+
+.wsp {
+  color: rgb(42, 42, 122);
+  font-size: 20px;
+}
+
+#projects {
+  text-align: center;
+  background-color: rgb(160, 84, 160);
+}
+
+.projects-header {
+  padding: 30px 0px 25px 0px;
```

Comment:
In order to maintain the best CSS practice, I think would be better to avoid adding zero units  😊. 
You can as well re-write this code by using just padding-top and padding-bottom. 👍🏼

Reply:
- [ ] Nicely done